src/main/java/com/hubspot/jinjava/JinjavaConfig.java

```diff
@@ -125,6 +128,10 @@ public boolean isFailOnUnknownTokens() {
     return failOnUnknownTokens;
   }
 
+  public boolean isEnableNestedInterpretation() {
```

Comment:
if the boolean was called `nestedInterpretationEnabled`, then the getter would read more cleanly as `isNestedInterpretationEnabled`